packages/shared/src/components/MainLayout.tsx

```diff
@@ -34,10 +35,17 @@ import {
   useActiveFeedNameContext,
 } from '../contexts';
 import { useFeedLayout, useViewSize, ViewSize } from '../hooks';
-import { GoBackHeaderMobile } from './post/GoBackHeaderMobile';
 import { BootPopups } from './modals/BootPopups';
 import { useFeedName } from '../hooks/feed/useFeedName';
 
+const GoBackHeaderMobile = dynamic(
```

Comment:
I think you can revert this with the latest change again as it's always rendered?

Reply:
But it's content is dynamic and not static on build time.